src/website/views/components/table/_common.pug

```diff
@@ -19,36 +19,74 @@ mixin tr(name, id, date)
     td= id
     td= date
 
-mixin table(row, column)
-  section.a-table&attributes(attributes)
-    div
-      table
-        thead
-          tr
-            th(class={
-              [`-sorted -${column}`]: column != undefined
-            })
-              div name
-            th
-              div ID
-            th
-              div last login
-        tbody
-          if column == 'ascending'
-            +tr('Manuel Martin', 'manuel', 'Nov 5, 2017 10:15 AM')
-            +tr('Diego Sanjuan', 'diego', 'Dec 18, 2017 2:38 AM')(
-              class={
-                [`-${row}`]: row != undefined && row != 'disabled'
-              }
-              disabled= row=='disabled' ? 'disabled' : false
-            )
-            +tr('David Navarro', 'david', 'Dec 18, 2017 3:26 PM')
+mixin table(row, column, selectable)
+  table.a-table&attributes(attributes)
+    thead
+      if !selectable
+        tr(class=(`-${row}`))
+          if column != undefined
+            if column == 'ascending'
+              th(class=(`-sorted -${column}`))
+                div name
+              th ID
+              th last login
+            else if column == 'descending'
+              th(class=(`-sorted -${column}`))
+                div name
+              th ID
+              th last login
           else
-            +tr('David Navarro', 'david', 'Dec 18, 2017 3:26 PM')
-            +tr('Diego Sanjuan', 'diego', 'Dec 18, 2017 2:38 AM')(
-              class={
-                [`-${row}`]: row != undefined && row != 'disabled'
-              }
-              disabled= row=='disabled' ? 'disabled' : false
-            )
-            +tr('Manuel Martin', 'manuel', 'Nov 5, 2017 10:15 AM')
+            th name
+            th ID
+            th last login
+      else if selectable
+        tr
+          if column != undefined
```

Comment:
@davidojedacl this is adding a -undefined class in the tr's in thead. Can you remove it?